src/python/scripts/dx-docker

```diff
@@ -271,35 +271,15 @@ def run(args):
         bounded_dirs.append(b)
 
     # TODO: implement some form of nested subprocesses to avoid quoting command array if possible
-    #proot_cmd =  ["proot", '-r', '{tmpdir}/rootfs'.format(tmpdir=tmpdir)] + \
-    #             bounded_dirs + \
-    #             volume + \
-    #             ["-w", workdir] + \
-    #             env + ["/bin/sh", "-c"] + \
-    #             [" ".join(("'"+x+"'" for x in container_cmd))]
-
-
-    tempsh_file_descriptor, tempsh_file_path = tempfile.mkstemp(suffix='.sh')
-    with os.fdopen(tempsh_file_descriptor, 'w') as tmpsh:
-        lines = ['#!/bin/sh']
-        lines.extend(env[2:])
-        lines.append(subprocess.list2cmdline(container_cmd))
-        tmpsh.write('\n'.join(lines))
-
-    os.chmod(tempsh_file_path, 0o777)
-
-
-    # Note that we assume /dx-docker-runme.sh doesn't already exist in the container
-    bounded_dirs.extend(['-b', tempsh_file_path + ':/dx-docker-runme.sh'])
-
     proot_cmd =  ["proot", '-r', '{tmpdir}/rootfs'.format(tmpdir=tmpdir)] + \
                  bounded_dirs + \
                  volume + \
                  ["-w", workdir] + \
-                 ['/dx-docker-runme.sh']
+                 env + ["/bin/sh", "-c"] + \
+                 [subprocess.list2cmdline(container_cmd)]
```

Comment:
Correct me if I'm wrong: if container_cmd is an empty list (entrypoint & cmd were not set), then this will not error out?